templates/includes/sidebar.html

```diff
@@ -0,0 +1,6 @@
+<ul>
+    <li><a href="#">Home page</a></li>
+    <li><a href="#">Manufacturers</a></li>
+    <li><a href="#">Cars</a></li>
+    <li><a href="#">Drivers</a></li>
```

Comment:
The href attributes in the anchor tags are currently set to '#', which means they are placeholders. You should replace them with actual URLs or Django URL template tags to ensure proper navigation.